library/src/main/java/com/mikepenz/aboutlibraries/LibsBuilder.kt

```diff
@@ -374,6 +375,18 @@ class LibsBuilder : Serializable {
         return this
```

Comment:
Up to you, but in general you can make these variables public and allow users to modify it via `.apply`. Builders may be redundant in that situation

Reply:
this is very much still legacy code :/